README.md

```diff
@@ -41,28 +41,28 @@ sktime is a library for time series analysis in Python. It provides a unified in
 [roadmap]: https://www.sktime.net/en/latest/roadmap.html
 [related software]: https://www.sktime.net/en/latest/related_software.html
 
-## :speech_balloon: Where to ask questions
+## :speech_balloon: Where to ask questions!
```

Comment:
I think the "!" doesn't work here

Reply:
ok removing that.

Reply:
what about  :eyes: this instead of that (not necessary, just asking casually😹)